main/pom.xml

```diff
@@ -35,8 +35,9 @@
 			<url>http://nexus.hc.to/content/repositories/pub_releases</url>
 		</repository>
 		<repository>
-			<id>bstats-repo</id>
-			<url>https://repo.bstats.org/content/repositories/releases/</url>
+			<!-- for bstats -->
+			<id>jitpack-repo</id>
+			<url>hhttps://jitpack.io/</url>

```

Comment:
^ URL line is wrong too (though as mentioned in my PR comment, jitpack is best to avoid entirely anyway)

Reply:
thanks, fixed